src/flow-manager.c

```diff
@@ -254,6 +259,54 @@ static int FlowManagerFlowTimeout(const Flow *f, enum FlowState state, struct ti
     return 1;
 }
 
+static inline int FlowBypassedTimeout(Flow *f, struct timeval *ts,
+                                      FlowTimeoutCounters *counters)
```

Comment:
I don't think counters here should be optional. AFAICS it will only be called with NULL from some unittests. So those should just also pass counters.

Reply:
I don't understand your comment.